app/enterprise/0.32-x/oidc-auth0.md

```diff
@@ -44,5 +44,5 @@ For basic authentication, use your client ID as the username and your client sec
 [client-credentials-grant]: https://auth0.com/docs/api-auth/tutorials/client-credentials
 [create-auth0-api]: https://auth0.com/docs/apis#how-to-configure-an-api-in-auth0
 [non-interactive-client]: https://auth0.com/docs/clients
-[create-api]: /latest/admin-api/#add-api
+[create-api]: /0.13.x/admin-api/#add-api
```

Comment:
This doesn't appear on the preview page.

Reply:
https://docs.konghq.com/0.13.x/admin-api/#add-api